### DISCUSSION TITLE
How to move git icons to the last column?

### DISCUSSION THREAD

**Discussion by Nikit-Singh** [nvim-tree/nvim-tree.lua#1335] [Q&A] [answered]
I wanted to know how can I move the git icon to the end of line? When I use `git_placement: 'after'` config, it just moves the symbol right next to the filename and not at the end of the line (which I know is the current intended behavior). So is there any setting or workaround for this?

Thanks for this great plugin BTW :)

**Comment by alex-courtis** [comment, marked as answer]
That functionality is not available. It could be added.

PRs are always welcome.

Please note that this change would be a non-trivial amount of work.

**Comment by Nikit-Singh** [comment]
Hi, thanks for the response. I'm pretty new to the vim ecosystem, but will try to get this feature done :wrench:.

**Reply by kyazdani42** [reply]
Note that this involves quite a huge amount of UX complications, since "the end of the buffer" can mean very much different things, filename can overflow, buffer size can change etcetc...

**Reply by Nikit-Singh** [reply]
Understood. I will get this done (albeit slowly :p). You folks can later help me with the testing and I can get all the edge cases fixed at that moment.
